lib/generators/draft/resource/templates/specs/factories.rb

```diff
@@ -18,16 +16,14 @@
     <%= attribute.column_name %> Time.now
 <% when :decimal -%>
     <%= attribute.column_name %> 42.42
-<% when :integer, :primary_key -%>
-    sequence(:<%= attribute.column_name %>, IncreasingRandom.new) { |n| n.value }
 <% when :string, :text -%>
-    sequence(:<%= attribute.column_name %>, IncreasingRandom.new) { |n| "Some fake <%= attribute.human_name.downcase %> #{n}" }
+    <%= attribute.column_name %> "Some fake <%= attribute.human_name.downcase %>"
```

Comment:
The goal is to replace `IncreasingRandom` with [`Faker`'s unique values](https://github.com/stympy/faker#ensuring-unique-values) eventually, but even before that we need to keep some unique element in these for the tests not to pass by accident. We can just use the [built-in sequence of integers](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences) for the time being.